contributing/testing-best-practices.md

```diff
@@ -0,0 +1,50 @@
+
+## Testing Best Practices
+
+### Types of test
```

Comment:
```suggestion
# Testing Best Practices

## Types of test
```
a11y preserve the title order and don't skip levels

Reply:
i see, thanks, i was following the other contributing files.    changed, will change the other files in a different PR